Sources/Shared/ImageManager.swift

```diff
@@ -69,7 +71,10 @@ public class ImageManager {
       case let .success(image, bytes):
         Configuration.track?(url, nil, bytes)
         if useCache {
-          Configuration.imageCache.async.addObject(image, forKey: url.absoluteString)
+          let wrapper = ImageWrapper(image: image)
+          Configuration.imageStorage?.async.setObject(wrapper, forKey: url.absoluteString, expiry: nil) { _ in
```

Comment:
Shouldn't completion be optional?

Reply:
@vadymmarkov: no, this callback can't be both `@escaping` and optional

Reply:
But from public API perspective it's not very nice to always have completion.